EIPS/eip-7045.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 7045
+title: Increase max attestation inclusion slot
+description: Increases max attestaton inclusion slot to the last slot in `N+1` where `N` is the epoch containing the attestations slot.
+author: Danny Ryan (@djrtwo)
+discussions-to: https://ethereum-magicians.org/t/eip-7045-increase-attestation-slot-inclusion-range/14342
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-05-18
+---
+
+## Abstract
+
+Increases max attestaton inclusion slot from `attestation.slot + SLOTS_PER_EPOCH` the last slot in `N+1` where `N` is the epoch containing the attestations slot.
+
+This increase is critical to the current LMD-GHOST security analysis as well as the confirmation rule.
+
+## Motivation
+
+Attestations can currently be included after some minimum delay (`1` slot on mainnet) up until `SLOTS_PER_EPOCH` slots after the slot the attestation was created in. This rolling window of one epoch was decided upon during Phase 0 because the equal inclusion window for any attestation was assessed as "fair". The alternative considered path was to allow inclusion during the current and next epoch which means attestations created during the start of an epoch have more potential slots of inclusion than those at the end of the epoch.
+
+Since this decision was initially made, it has become apparent that the alternative design is critical for current LMD-GHOST security proofs as well as a new confirmation rule (which will allow for block confirmations in approximately 3-4 slots in normal mainnet conditions).
```

Comment:
```suggestion
Since this decision, it has become apparent that the alternative design is critical for current LMD-GHOST security proofs as well as a new confirmation rule (which will allow for block confirmations in approximately 3-4 slots in normal mainnet conditions).
```